lib/shortcodes/login.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+function ms_login() {
```

Comment:
Login will be entirely in React since all UI of URLsLab will be in React, including the sign in and sign up screen.

Reply:
Removed